src/rlookup.c

```diff
@@ -581,7 +581,7 @@ static int getKeyCommonHash(const RLookupKey *kk, RLookupRow *dst, RLookupLoadOp
     // as it will hold the only reference to it after the next line.
     rsv = hvalToValue(val, kk->fieldtype);
     RedisModule_FreeString(RSDummyContext, val);
-  } else if (!strncmp(kk->name, UNDERSCORE_KEY, strlen(UNDERSCORE_KEY))) {
+  } else if (!strncmp(kk->path, UNDERSCORE_KEY, strlen(UNDERSCORE_KEY))) {
```

Comment:
should we make `strlen(UNDERSCORE_KEY)` a static const or just rely on intelligence of optimizer?

Reply:
I want to believe that `-O3` and even `-O2` are doing so